src/components/Pega_Extensions_QRCode/styles.ts

```diff
@@ -0,0 +1,13 @@
+import styled, { css } from 'styled-components';
+
+export default styled.div(() => {
+  return css`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  img {
+    max-width: 150px;
```

Comment:
do not use px width - use rem - should height passed as parameter similar to barcode?